esphome/components/bl0940/bl0940.h

```diff
@@ -84,26 +114,74 @@ class BL0940 : public PollingComponent, public uart::UARTDevice {
   sensor::Sensor *internal_temperature_sensor_{nullptr};
   sensor::Sensor *external_temperature_sensor_{nullptr};
 
+#ifdef USE_NUMBER
+  // calibration helpers
+  number::Number *voltage_calibration_{nullptr};
+  number::Number *current_calibration_{nullptr};
+  number::Number *power_calibration_{nullptr};
+  number::Number *energy_calibration_{nullptr};
```

Comment:
Best to be explicit, and matches the sensors above and button below.
```suggestion
  number::Number *voltage_calibration_number_{nullptr};
  number::Number *current_calibration_number_{nullptr};
  number::Number *power_calibration_number_{nullptr};
  number::Number *energy_calibration_number_{nullptr};
```